cli/src/commands/install.js

```diff
@@ -118,23 +118,24 @@ export async function run(args: Args): Promise<number> {
   // error caused by rebasing the libdedf cache concurrently.
   const defs: Array<LibDef> = [];
   for(let dep in depsMap) {
+    const depVersion = depsMap[dep]; 
     let def = await findLibDef(
       dep,
-      depsMap[dep],
+      depVersion,
       flowVersion);
     if(def) {
       defs.push(def);
+      if(libdefNeedsUpdate(def.pkgVersionStr, depVersion)) {
+        console.log(`${def.pkgName}_${def.pkgVersionStr} may need an update to fully support ${dep} at ${depVersion}.`);
```

Comment:
Suggestion:

`Installed ${def.pkgName}_${def.pkgVersionStr}, but you may want to create an updated libdef in flow-typed for your version (${depVersion}).`